Getting_Started.md

```diff
@@ -0,0 +1,128 @@
+# Getting Started with Bunsen
+
+This guide is intended to help you get your first Bunsen environment up and running.
+
+# Expectations
+
+This project is simply an ansible playbook that has a certain set of requirements that need to be satisfied moving in.
+
+## Initial requirements
+
+Before you can start provisioning your environment, the following items must be satisfied:
+
+1. An environment must consist of at least 3 different systems (physical or virtual).  It is recommended that you have at least 5 machines, but minimally 3 machines can work.  More information on what the different systems will be doing will be provided later.
+    - Of those 3 machines, one must have at least 280GiB of additional storage in a separate block device (i.e. sdb)
+2. Each machine must be able to communicate with the others in the environment both by IP address and by name (i.e. ossbunsen-resource [10.2.0.5] needs to be able to ssh to ossbunsen-farm-f36-1 [10.2.0.6])
+3. The provisioning user must have an ssh-key generated and located under `~/.ssh`.
+4. A `controller` system that has ansible installed on it.
+
+You can find an example environment that utilizes vagrant and libvirt at [the VDO vagrant example repository](https://github.com/dm-vdo/vagrant)
+
+## System Functions
+
+A provisioned Bunsen Environment consists of three different types of machine.
+
+- Infrastructure
+    - This machine will provide necessary infrastructure services for the other systems to utilize.
+        - Such as NFS, [RSVP](https://github.com/dm-vdo/permabit-rsvpd) (resource management and coordination), iSCSI backing storage (if applcable)
+- Resource
+    - General development and compiling will be done on this machine.
+    - If there are multiple incompatible targets (i.e. Fedora, RHEL), then one of each possible target should be available to ensure that the compilations are done according to their environment.
+- Farm
+    - At least one of each applicable targets (i.e. Fedora, RHEL).
+        - With Fedora, we typically utilize at least one of each supported version of Fedora in case there is a change applied to one and not the other.
+    - Each farm must have an additional disk (i.e. sdb) with 280 GiB or more of storage to use as scratch space for testing.
+- Controller
+    - This system can be the hypervisor that is running the various VMs.  Or it could be an additional VM living next to the others.
+
+# Deploying your first environment
+
+Assuming that the dependencies above have been satisfied, you can now provision a Bunsen environment.  In order to deploy a bunsen environment, you need a few things first.
```

Comment:
```suggestion
Assuming that the dependencies above have been satisfied, you can now provision a Bunsen environment.  In order to deploy a bunsen environment, you need the following...
```

Reply:
A form of this will be in my next rev.  (probably without the trailing ...'s)

Reply:
This is addressed in my recent pushes.